README.md

```diff
@@ -1 +1,44 @@
-# GitHub-Flow
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> In this project, you will set up a "Hello world" repository. No complex coding is required for this exercise. Your goal here is to master all of the tools and best practices you learned about in previous steps. You will be using them in all Microverse projects and most likely in your future job as well, so it is important to know them!
+
+
+## Built With
+
+- HTML, CSS
+- VS Code
+
+
+
+## Getting Started
+
+
+
```

Comment:
- [ ] [OPTIONAL] In my opinion, it would be better if you could provide information on how to run your project in this section 👍